pyproject.toml

```diff
@@ -1,5 +1,42 @@
 # SPDX-License-Identifier: MIT
 
+[build-system]
+requires = ["setuptools>61"]
```

Comment:
No, we only require setuptools to build the package.

Reply:
`wheel` is required too

Reply:
https://peps.python.org/pep-0518/#build-system-table

Reply:
Interesting, reading the setuptools documentation implied that it was not required. https://setuptools.pypa.io/en/latest/build_meta.html#how-to-use-it

Reply:
After reading through a bit of setuptools source code, and reading PEP 517 a bit closer, it turns out that setuptools backend provides wheel in their implementation of [get-requires-for-build-wheel](https://peps.python.org/pep-0517/#get-requires-for-build-wheel). Because `wheel` isn't required for building an sdist, I don't think we need to declare wheel here, but is there any downside to declaring it?